Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocIndexColumnRenderer.cs

```diff
@@ -100,16 +102,16 @@ public void RenderTitleCell (BocListRenderingContext renderingContext)
       if (_renderingFeatures.EnableDiagnosticMetadata)
       {
         var columnTitle = renderingContext.Control.IndexColumnTitle;
-        if (!string.IsNullOrEmpty (columnTitle))
-          renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle));
+        if (!columnTitle.IsEmpty)
+          renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle.GetValue()));
 
         AddDiagnosticMetadataListCellIndex (renderingContext);
       }
       renderingContext.Writer.RenderBeginTag (HtmlTextWriterTag.Th);
       renderingContext.Writer.RenderBeginTag (HtmlTextWriterTag.Span);
-      string indexColumnTitle = renderingContext.Control.IndexColumnTitle;
-      if (string.IsNullOrEmpty (renderingContext.Control.IndexColumnTitle))
-        indexColumnTitle = renderingContext.Control.GetResourceManager().GetString (BocList.ResourceIdentifier.IndexColumnTitle);
+      WebString indexColumnTitle = renderingContext.Control.IndexColumnTitle;
+      if (indexColumnTitle.IsEmpty)
+        indexColumnTitle = renderingContext.Control.GetResourceManager().GetWebString (BocList.ResourceIdentifier.IndexColumnTitle, indexColumnTitle.Type);
```

Comment:
```suggestion
      WebString indexColumnTitle = renderingContext.Control.IndexColumnTitle;
      if (indexColumnTitle.IsEmpty)
        indexColumnTitle = renderingContext.Control.GetResourceManager().GetText (BocList.ResourceIdentifier.IndexColumnTitle);
```

We only use the Type if the Value is used to resolve the resource. otherwise, our code defines the resource's type